demo/src/test/java/com/example/test/pages/KBBMOOCProject/WhatMyCarWorthPage/MyCarValuePage.java

```diff
@@ -0,0 +1,86 @@
+package com.example.test.pages.KBBMOOCProject.WhatMyCarWorthPage;
```

Comment:
should write the web element locator in separate line then call into a method/function

Reply:
Write web element locator in separate line then call into a method/function